phpcs.xml.dist

```diff
@@ -21,6 +21,10 @@
         <exclude name="SlevomatCodingStandard.Classes.DisallowLateStaticBindingForConstants.DisallowedLateStaticBindingForConstant"/>
     </rule>
 
+    <rule ref="Squiz.NamingConventions.ValidVariableName.PublicHasUnderscore">
```

Comment:
Did this rule change its name? It must have been already ignored.

Reply:
Doesn't look like it. I checked and couldn't find a rule with a similar name, and besides, if the name changes, phpcs refuses to run because it does not recognize the old name.

Reply:
Why are we only introducing the suppression right now? Wasn't this requirement already part of the standard?

Reply:
I guess not: when I checkout `origin/2.10.x`, remove all the exclude rules from `phpcs.xml.dist`, and run phpcs against `lib/Doctrine/DBAL/Event/Listeners/OracleSessionInit.php`, which is an offender for this rule, I get no complaints.

Reply:
@morozov maybe you are thinking of this: https://github.com/doctrine/coding-standard/blob/c665baebb119e087b421e282a8565519be4c6c00/lib/Doctrine/ruleset.xml#L486-L488

The name of this one suggests it's about private properties only.

Reply:
Oh, I didn't realize there are different rules for different types of visibility. Thank you.